src/status_im/contexts/wallet/common/activity_tab/events.cljs

```diff
@@ -1,70 +1,132 @@
 (ns status-im.contexts.wallet.common.activity-tab.events
   (:require [camel-snake-kebab.extras :as cske]
+            [utils.collection :as collection]
             [utils.ethereum.chain :as chain]
             [utils.re-frame :as rf]
             [utils.transforms :as transforms]))
 
-(defonce ^:private request-id-atom (atom 0))
+;; Constants
```

Comment:
I feel these comments are unnecessary, it is implicit which declarations are constants, helper functions or event handlers